pkgs/applications/misc/lswt/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "v1.0.4";
```

versions should not include `v` prefix (this does mean you will need to use `"v${version}" below for `rev`)
```suggestion
  version = "1.0.4";
```